src/main/java/com/hivemq/cli/commands/hivemq/datagovernance/DataGovernanceOptions.java

```diff
@@ -0,0 +1,54 @@
+package com.hivemq.cli.commands.hivemq.datagovernance;
+
+import com.hivemq.cli.utils.LoggerUtils;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.VisibleForTesting;
+import picocli.CommandLine;
+
+public class DataGovernanceOptions {
+
+    @SuppressWarnings({"NotNullFieldNotInitialized", "unused"}) // will be initialized via default value
+    @CommandLine.Option(names = {"-u", "--url"},
+                        defaultValue = "http://localhost:8888",
+                        description = "The URL of the HiveMQ REST API endpoint (default http://localhost:8888)",
+                        order = 1)
+    private @NotNull String url;
+
+    @SuppressWarnings({"unused"})
+    @CommandLine.Option(names = {"-r", "--rate"},
+                        defaultValue = "1500",
+                        description = "The rate limit of the REST calls to the HiveMQ API endpoint in requests per second (default 1500/s)",
+                        order = 3)
```

Comment:
Could probably be given the `order = 2`, and the following `initLogging` could be given the `order =3` to make it consistent with the `ExportClientsCommand`.